pkg/controller/command/command_upgrade.go

```diff
@@ -76,6 +95,19 @@ func getContainerImage(containers []core.Container, name string) string {
 	return ""
 }
 
+func (r *ReconcileCommand) checkIfImageChangedAgain(command *contrail.Command) (bool, error) {
+	dataMigrationJob := &batch.Job{}
+	jobName := types.NamespacedName{Namespace: command.Namespace, Name: command.Name + "-upgrade-job"}
+	err := r.client.Get(context.Background(), jobName, dataMigrationJob)
+	if errors.IsNotFound(err) {
+		return false, nil
+	}
+	if err != nil {
+		return false, err
+	}
+	return dataMigrationJob.Spec.Template.Spec.Containers[0].Image != getImage(command.Spec.ServiceConfiguration.Containers, "api"), nil
+}
+
 func (r *ReconcileCommand) checkDataMigrationJobStatus(command *contrail.Command) error {
 	dataMigrationJob := &batch.Job{}
 	jobName := types.NamespacedName{Namespace: command.Namespace, Name: command.Name + "-upgrade-job"}
```

Comment:
Should this "check" method create migrationJob `r.client.Create(context.Background(), dataMigrationJob)`

Reply:
no it should not, copy paste mistake